### PR TITLE
Starting Java language bootcamp

### PR DESCRIPTION
**Pull request by smigii**
Started section 1 and 2, basics and control flow.

### CODE REVIEW

File: docs/language_bootcamp/java/2_ControlFlow/ifelse.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+public class ifelse {
```
Comment (robguderian) [inline docs/language_bootcamp/java/2_ControlFlow/ifelse.java:1]: Tisk tisk!

**Comment by robguderian** [comment]
Super awesome. Install a markdown linter in whatever IDE you have, and clear the errors (spaces around code fences, after ## things. When you do another PR, it'll update this one.

I have the one on a line (mostly for me to try it) that is about best practices. We need to be picky about that.

You'll still want the `readme.md` in the root, at least as a table of contents.

**Comment by robguderian** [comment]
Amazing work.

Get rid of the class files

image


Add a `.gitignore` in the Java directory to ignore all `.class` files for the future, and `add` the `.gitignore` you have to the repo.

Otherwise, awesome. I'm sure there will be edits as we go on, but awesome.

**Comment by smigii** [comment]
So those 2 class files I explicitly added in there.

For the final challenge of section 2, I wanted the students to basically be working with a "blackbox". My thought is that

* It would feel more rewarding when they finally break in
* They would have no way of cheating (well, no easy way)

I compiled those on the rodents machines and put in the instructions what version they need (java 8 or up) to run the class files.

Thoughts? I could just put the source for Server and Client in the repository, but I thought this was worth a try.

**Pull request merged**